File: src/vehicle/rental/system/rental/Vehicle.java
package vehicle.rental.system.rental;

import java.math.BigDecimal;

public class Vehicle {
    private final String brand;
    private final String model;
    private final BigDecimal value;
    private final String type;

    public Vehicle(String brand, String model, BigDecimal value, String type) {
        this.brand = brand;
        this.model = model;
        this.value = value;
        this.type = type;
    }

    public String getBrand() {
        return brand;
    }

    public String getModel() {
        return model;
    }

    public BigDecimal getValue() {
        return value;
    }

    public String getType() {
        return type;
    }

}

File: src/Rental/Vehicle.cs
namespace VehicleRentalSystem.Rental;

public class Vehicle
{
    public string Brand { get; }
    public string Model { get; }
    public decimal Value { get; }
    public string Type { get; }

    public Vehicle(string brand, string model, decimal value, string type)
    {
        Brand = brand;
        Model = model;
        Value = value;
        Type = type;
    }
}
